cheeseshop/dbapi.py

```diff
@@ -323,8 +392,103 @@ class CsGoDeathEvent(object):
             CREATE TABLE cs_go_death_events(
                 id serial PRIMARY KEY,
                 attacker integer REFERENCES cs_go_steam_ids (id),
+                attacker_pos_x real,
```

Comment:
I get the motivation with adding all of these properties, but I would recommend starting with the minimal set. The idea of the separate hltv events table is it should be trivial for us to add in properties later on and if we start huge like this then theres just going to be a lot of initial work before getting an end-to-end demo working.

Reply:
Yah I agree that anything not directly needed should be dropped. All of these are pulled directly from the example paste ryan put together of what's already in the death event. If any of these values give us trouble, drop them then?